zzz_generated/placeholder/app1/zap-generated/CHIPClientCallbacks.h

```diff
@@ -30,6 +30,12 @@
 #include <lib/support/Span.h>
 
 // List specific responses
+typedef void (*OnOffGeneratedCommandListListAttributeCallback)(void * context,
+                                                               const chip::app::DataModel::DecodableList<chip::CommandId> & data);
+typedef void (*OnOffAcceptedCommandListListAttributeCallback)(void * context,
+                                                              const chip::app::DataModel::DecodableList<chip::CommandId> & data);
+typedef void (*OnOffAttributeListListAttributeCallback)(void * context,
+                                                        const chip::app::DataModel::DecodableList<chip::AttributeId> & data);
```

Comment:
Why are these being added?

Reply:
Updated this as well. Now if the server side cluster/attributes are not enabled for the client cluster then these are not generated.